src/javaRosa.js

```diff
@@ -1980,6 +1973,24 @@ define([
                 widget: iTextIDWidget,
                 widgetValuePath: "constraintMsgItext"
             };
+            databind.constraintMediaIText = function (mugOptions) {
+                return mugOptions.isSpecialGroup ? undefined : {
```

Comment:
Ignore my previous comment. It seems safe to say all browsers we target are ES5 compliant, and therefore `undefined` is immutable. I guess that means we can stop using `_.isUndefined(x)` in favor of `x === undefined`, which I like much better. http://stackoverflow.com/a/8783528/10840